src/components/v2/Select/index.tsx

```diff
@@ -0,0 +1,104 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState, useMemo } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+import { SelectChangeEvent } from '@mui/material/Select';
+import { MenuProps } from '@mui/material/Menu';
+import { Select as MuiSelect } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import { useIsSmDown } from 'hooks/responsive';
+import { Icon } from '../Icon';
+import { TextButton } from '../Button';
+import { SELECTED_MENU_ITEM_CLASSNAME, useStyles } from './styles';
+
+interface IOption {
+  value: string;
+  label: string;
+}
+
+export interface ISelectProps {
+  className?: string;
+  options: IOption[];
+  value: string | undefined;
+  onChange: (e: SelectChangeEvent) => void;
+  ariaLabel: string;
+  title: string;
+}
+
+export const Select: React.FC<ISelectProps> = ({
+  className,
+  options,
+  value,
+  onChange,
+  ariaLabel,
+  title,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const styles = useStyles();
+  const isSmDown = useIsSmDown();
+
+  const handleClose = () => {
+    setIsOpen(false);
+  };
+
+  const handleOpen = () => {
+    setIsOpen(true);
+  };
+
+  const menuProps = useMemo(() => {
+    const mobileStyles: Partial<MenuProps> = {
+      transformOrigin: {
+        vertical: 'top',
+        horizontal: 'center',
+      },
+      anchorReference: 'none',
+      anchorOrigin: {
+        vertical: 'top',
+        horizontal: 'center',
+      },
+    };
+    return {
+      PaperProps: {
+        sx: styles.menuWrapper,
+      },
+      ...(isSmDown ? mobileStyles : {}),
```

Comment:
Actually, I just saw that.

We want to move away from using JS for responsiveness: let's use CSS directly here (you can render both mobile and desktop instances of the component and choose which one to render based on the screen size using CSS directly).
Also, please don't copy and paste the whole code: you can make it so only the style prop differs from one version to the other.